src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -172,10 +143,15 @@ export default class HaAutomationActionRow extends LitElement {
               )}
             </div>`
           : ""}
-        <div class="card-menu">
+        <ha-expansion-panel leftChevron>
+          <div slot="header" class="name">
+            ${describeAction(this.hass, this.action)}
+          </div>
+
           ${this.index !== 0
             ? html`
                 <ha-icon-button
+                  slot="header"
```

Comment:
Can't do this.  The entire header is already a button, so now you have a button inside a button which causes lots of problems.

I like the idea of being able to move them up and down without expanding (which is what I assume this does? I can't operate it.).  I think you need to create another optional slot in `ha-expansion-panel` to pass this in.